polyphemus/lib/etls/add_watch_folder_base_etl.rb

```diff
@@ -1,7 +1,8 @@
 require_relative "./metis_folder_filtering_base_etl"
 
 class Polyphemus::AddWatchFolderBaseEtl < Polyphemus::MetisFolderFilteringBaseEtl
-  def initialize(project_bucket_pairs:, model_name:, folder_path_regexes: {}, limit: 20, watch_type:)
+  def initialize(project_bucket_pairs:, model_name:, folder_path_regexes: {}, limit: 20, watch_type:, linker:)
```

Comment:
I felt a little better about making this explicit in the constructor for now, but to be honest I plan to do another refactoring with my general ETL changes that make the folder watching etl work a little more robust and less coupled directly to the linker.